### ISSUE TITLE
Add clause for 'Specifications using JSON schema to state requirements'

### ISSUE THREAD

**Issue by dr-shorthair** [opengeospatial/ogc-modspec#17]
Alongside the clauses for UML, XML Schema etc. 

Should make some comments or recommendations on the _version_ of JSON Schema to be used.

**Comment by cnreediii** [comment]
@dr-shorthair Yes, this recommendation is in the slide set I presented at the Montreal closing plenary.
